ci/headers.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2017 DigitalOcean
```

Comment:
I think the biggest challenge with the header checks is that the format varies depending on the source file. This check will work for Go, but it will change for bash scripts, Dockerfiles, etc. I wouldn't be opposed to having a list of known formats and just using that. What do you think?

Reply:
Do you want me to create bash scripts for several known such as Bash and Dockerfiles or you have better idea?

Reply:
I think several known is probably the best for now.